thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DashboardResource.java

```diff
@@ -558,4 +565,29 @@ public String getData(@QueryParam("type") String type) throws Exception {
     }
     return response;
   }
+
+  @POST
+  @Path("/severity")
+  public Response computeSeverity(@NotNull @QueryParam("collection") String collectionName,
+      @NotNull @QueryParam("metric") String metricName,
+      @NotNull @QueryParam("start") String startTimeIso, @NotNull @QueryParam("end") String endTimeIso)
+      throws Exception {
+    DateTime startTime = null;
+    DateTime endTime = null;
+    if (StringUtils.isNotBlank(startTimeIso)) {
+      startTime = ISODateTimeFormat.dateTimeParser().parseDateTime(startTimeIso);
+    }
+    if (StringUtils.isNotBlank(endTimeIso)) {
+      endTime = ISODateTimeFormat.dateTimeParser().parseDateTime(endTimeIso);
+    }
+    PinotThirdEyeClient thirdEyeClient = PinotThirdEyeClient.getDefaultTestClient();
```

Comment:
this was for testing. You should probably get the thirdeyePinotClient from else where or pass it when resource is getting created